src/persist-client/src/batch.rs

```diff
@@ -684,9 +684,12 @@ pub(crate) struct BatchParts<T> {
 }
 
 fn force_keep_stats_col(name: &str) -> bool {
-    name == "mz_internal_super_secret_source_data_errors"
-        || name == "timestamp"
+    // If we trim the "err" column, then we can't ever use pushdown on this part
+    // (because it could have >0 errors).
+    name == "err"
+        // Various flavors of timestamp column names found in the wild.
         || name == "ts"
+        || name.ends_with("timestamp")
```

Comment:
We've seen at least one `Timestamp` in the wild -- note the casing. Might be worth relaxing this slightly or adding another case?

Reply:
Hmm... I think the tests prove that the names are downcased before they make it here, in which case disregard!

Reply:
Yup, they are downcased. I'll toss in a comment